server/src/Controller/User/UserController.js

```diff
@@ -0,0 +1,80 @@
+import Repository from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { userModel } from '../../Model';
+
+const repository = new Repository(userModel);
```

Comment:
Try to reduce the scope of variable as much as possible.

Reply:
Resolved as I have removed the class and created the separate functions.